src/client/cmd/connect.cpp

```diff
@@ -45,8 +45,16 @@ mp::ReturnCode cmd::Connect::run(mp::ArgParser* parser)
             return ReturnCode::Ok;
 
         auto priv_key_blob = reply.priv_key_base64();
-        mp::SSHClient ssh_client{host, port, priv_key_blob};
-        ssh_client.connect();
+        try
+        {
+            mp::SSHClient ssh_client{host, port, priv_key_blob};
+            ssh_client.connect();
+        }
+        catch (const std::exception& e)
+        {
+            cerr << "connect failed: " << e.what() << std::endl;
```

Comment:
Think we can give better messages here, at least for some common scenarios?

Reply:
The what() should give a pretty good description about what the error is.  In the case that popey put in the message would be:
connect failed: ssh: No route to host

Other ssh issues will have different messages.  Is that good enough?

Reply:
It's coming from libssh via ssh_get_error().  The "ssh:" part of this is something we add when throwing the runtime_error.  See include/multipass/ssh/throw_on_error.h for more detail.  I'm really not sure how much more we can do without redoing a bunch of stuff...

Reply:
I guess I could just put in something like, "Can't connect to the instance" for any errors and be done with it?